src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -356,6 +357,13 @@ pub struct Args {
         requires = "storage-host-sizes"
     )]
     default_storage_host_size: Option<String>,
+    /// The interval in seconds at which to collect storage usage information.
+    #[clap(
+        long,
+        env = "STORAGE_USAGE_COLLECTION_INTERVAL_SEC",
+        default_value = "3600"
+    )]
+    storage_usage_collection_interval_sec: u64,
```

Comment:
Can you make this a `storage_usage_collection_interval: Duration` directly and tell clap to parse it using the [`parse_duration`](https://docs.rs/parse_duration/latest/parse_duration/) crate?